src/InvariantTheory/fundamental_invariants.jl

```diff
@@ -79,12 +77,12 @@ function fundamental_invariants_via_king(RG::InvRing, beta::Int = 0)
         continue
       end
 
-      _, g = divrem(f, GO)
+      g = reduce(f, GO, ordering = ordR)
       if is_zero(g)
         continue
       end
 
-      push!(S, inv(leading_coefficient(f))*f)
+      push!(S, inv(AbstractAlgebra.leading_coefficient(f))*f)
```

Comment:
Isn't this incompatible with the change to use `reduce`?

Reply:
Canceling the leading coefficient is mostly eye candy so that the polynomials look nicer, but there is no mathematical necessity. It should be done with the ordering that is used for the printing.

Reply:
Makes sense. Could you add a comment explaining this?